hardware/hetzner-cloud/default.nix

```diff
@@ -0,0 +1,29 @@
+{ modulesPath, ... }:
+{
+  imports = [
+    ../../profiles/cloud-init.nix
+    "${modulesPath}/profiles/qemu-guest.nix"
+  ];
+
+  config = {
+    boot.cleanTmpDir = true;
+    boot.growPartition = true;
+    boot.loader.grub.device = "/dev/sda";
+
+    fileSystems."/" = { device = "/dev/sda1"; fsType = "ext4"; };
+
+    networking.useNetworkd = true;
+    networking.useDHCP = false;
+
+    systemd.network.networks."10-uplink" = {
```

Comment:
An assertion to add an ipv6 address would be nice here. Otherwise we have servers with broken aaaa records again.

Reply:
added an assertion